src/lavinmq/exchange/fanout.cr

```diff
@@ -7,23 +7,27 @@ module LavinMQ
     end
 
     def bind(destination : Queue, routing_key, headers = nil)
-      @queue_bindings[{routing_key, nil}] << destination
+      ret = @queue_bindings[{routing_key, nil}].add? destination
       after_bind(destination, routing_key, headers)
+      ret
```

Comment:
Should `after_bind` be executed even if the binding was already there? 

```suggestion
      if @queue_bindings[{routing_key, nil}].add? destination
        after_bind(destination, routing_key, headers)
        true
      end
```

Reply:
Maybe, I just didn't want to change the behavior. Seems to be federation link that listens for bind/unbind events to forward them to the upstream.